README.md

```diff
@@ -430,6 +430,7 @@ Name | Website | Region
 [Nagarro](/company-profiles/nagarro.md) | https://www.nagarro.com/en | Worldwide
 [Nationwide](/company-profiles/nationwide.md) | https://www.nationwide.com/ | USA
 [NetApp](/company-profiles/netapp.md) | https://www.netapp.com/ | Worldwide
+[Netlify](/company-profiles/netlify.md) | https://app.netlify.com/ | USA 
```

Comment:
This is not in alphabetical order, it should be below `Netguru`

Reply:
Thanks for noticing it. Updated as you requested 